### PR TITLE
create Bookings controller, add routes

### PR DESCRIPTION
**Pull request by yourcontact**
@emanuelmoniz @RuiSMagalhaes 

1. I generated files with controller command in the terminal with references to user and toilet.
2. I populated the policy file with all method and left the create method as "true"; in the future we will have to update the method to make sure only user who have booked those toilets can leave a review.
3. I added the routes nested in toilets with create only.
4. In the controller I created the create method.  I tested the create method in Rail c and it works.

### CODE REVIEW

File: app/policies/review_policy.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+class ReviewPolicy < ApplicationPolicy
```
Comment (RuiSMagalhaes) [inline app/policies/review_policy.rb]: Not ReviewPolicy ! It's a BookingPolicy! 
Don't forget to change the file name to booking_policy.

File: app/policies/review_policy.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  def new?
+    create?
+  end
+
+  def create?
+    true
```
Comment (RuiSMagalhaes) [inline app/policies/review_policy.rb]: We only need the create? method

delete everything else

File: app/controllers/bookings_controller.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+class BookingsController < ApplicationController
+  def create
+    @toilet = Toilet.find(params[:toilet_id])
+    @booking = Booking.new
+    @booking.toilet = @toilet
+    @booking.user = current_user
+    @booking.save
+    redirect_to toilet_path
```
Comment (RuiSMagalhaes) [inline app/controllers/bookings_controller.rb]: add authorize @booking after line 8

File: app/views/bookings/toilet:references.html.erb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+<h1>Bookings#toilet:references</h1>
```
Comment (RuiSMagalhaes) [inline app/views/bookings/toilet:references.html.erb]: delete the file

File: app/views/bookings/user:references.html.erb [hunk 5]
```diff
@@ -0,0 +1,2 @@
+<h1>Bookings#user:references</h1>
```
Comment (RuiSMagalhaes) [inline app/views/bookings/user:references.html.erb]: delete the file

File: app/policies/review_policy.rb (hunk 1 above)
Comment (yourcontact) [reply app/policies/review_policy.rb]: @RuiSMagalhaes updated

File: app/policies/review_policy.rb (hunk 2 above)
Comment (yourcontact) [reply app/policies/review_policy.rb]: @RuiSMagalhaes done.
One quick question: what about when the booking expires? We mentioned that bookings would expires after 30 minutes, regardless.

Comment (yourcontact) [reply app/policies/review_policy.rb]: cc @emanuelmoniz @joaofrancosta

File: app/controllers/bookings_controller.rb (hunk 3 above)
Comment (yourcontact) [reply app/controllers/bookings_controller.rb]: @RuiSMagalhaes done

File: app/views/bookings/user:references.html.erb (hunk 5 above)
Comment (yourcontact) [reply app/views/bookings/user:references.html.erb]: done

File: app/views/bookings/toilet:references.html.erb (hunk 4 above)
Comment (yourcontact) [reply app/views/bookings/toilet:references.html.erb]: done

**Comment by RuiSMagalhaes** [comment]
@yourcontact i think you have to do a new pull request with the most recent commit.
by the way, in "bookings_controller.rb", line 8 is redirect_to @toilet

File: app/controllers/bookings_controller.rb (hunk 3 above)
Comment (RuiSMagalhaes) [reply app/controllers/bookings_controller.rb]: 1)you should put the authorize @booking before the redirect_to
2)line 8 shoud be redirect_to @toilet

**Pull request merged**